pyrometheus/__init__.py

```diff
@@ -360,9 +358,37 @@ def _my_make_array(res_list):
     return result
 
 
+def _pyro_norm(usr_np, argument, normord):
+    # Wrap norm for scalars
+    from numbers import Number
+    if isinstance(argument, Number):
+        return np.abs(argument)
+    return usr_np.linalg.norm(argument, normord)
+
+
 class Thermochemistry:
-    def __init__(self, npctx=np):
-        self.npctx = npctx
+    def __init__(self, usr_np=np):
+        \"""Initialize the mechanism object.
+
+        Parameters
+        ----------
+        usr_np
+            numpy-like namespace providing at least the following functions:
+            usr_np.log(T)
```

Comment:
It is difficult to understand that this is not uniquely tied to the temperature. Can we either (i) state we're using T and temperature as an example up front; or (ii) generalize and then provide T and temperature as an example? (Hoping this is clear, otherwise let me know.)